internal/handler/controller/domain.go

```diff
@@ -50,6 +94,9 @@ func (c *Controller) handleDomainCreate(w http.ResponseWriter, r *http.Request)
 		if !ok {
 			return nil, models.ErrUndefinedDomain
 		}
+		if c.Config.DomainVerification {
+			return nil, models.ErrDomainVerificationNotSupported
+		}
```

Comment:
If domain verification is disabled, it means verification is not required for custom domains.